db/migrate/20230607163814_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.references :author, foreign_key: { to_table: :users }
+      t.references :post, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Nice work so far 👏  However some errrors are happening when runing you migrations because some migrations are not well ordered and others are dupplicated

<img width="919" alt="image" src="https://github.com/ilyassatlassi/Blog-app/assets/65068771/b5298bb9-3de1-41ae-9558-63e6cf725790">

Tolve this have listered he only migrations you need and well orderd kindly make sure you create the you model in synchronization so that you may avoid referencing tables with are not yet created in your migrations.

Your migrations(with duplicted, and mal-ordered | Syncronized migrations(correct with no errors)
---|---
<img width="565" alt="image" src="https://github.com/ilyassatlassi/Blog-app/assets/65068771/2d7dfb42-f6ca-4207-b59c-8fc913df1a72"> | <img width="538" alt="image" src="https://github.com/ilyassatlassi/Blog-app/assets/65068771/06740821-d899-436e-9f07-3c64c2be40d2">
errors in  the terminal 👆  | no errors in the terminal 👆 
<img width="917" alt="image" src="https://github.com/ilyassatlassi/Blog-app/assets/65068771/c2b6abf6-f098-4a1b-97b1-ff7510f89463"> |  <img width="917" alt="image" src="https://github.com/ilyassatlassi/Blog-app/assets/65068771/ed382be8-1835-4cae-9dd8-87213d230a44">